cime_config/namelist_definition_drv.xml

```diff
@@ -2154,7 +2158,77 @@
     <type>integer</type>
     <category>aux_hist</category>
     <group>MED_attributes</group>
-    <desc>Auxiliary mediator wav2med file1 number of time samples per file. By default will be 30</desc>
+    <desc>Number of time samples per file.</desc>
+    <values>
+      <value>30</value>
+    </values>
+  </entry>
+
+   <!-- ======= -->
+
+  <entry id="histaux_wav2med_file2_enabled">
+    <type>logical</type>
+    <category>aux_hist</category>
+    <group>MED_attributes</group>
+    <desc>Auxiliary mediator wav2med average history output every day.</desc>
+    <values>
+      <value>.false.</value>
+    </values>
+  </entry>
+  <entry id="histaux_wav2med_file2_flds">
+    <type>char</type>
+    <category>aux_hist</category>
+    <group>MED_attributes</group>
+    <desc>Auxiliary mediator wav2med average history output every day.</desc>
+    <values>
+      <value>Sw_phs0:Sw_phs1:Sw_pdir0:Sw_pdir1:Sw_pTm10:Sw_pTm11</value>
+    </values>
+    <desc>Auxiliary mediator wav2med file2 output fields (colon delimited)</desc>
+  </entry>
+  <entry id="histaux_wav2med_file2_history_option">
+    <type>char</type>
+    <category>aux_hist</category>
+    <group>MED_attributes</group>
+    <desc>history option type</desc>
```

Comment:
This description is not really useful. I would rather either have a better id (e.g., `id="histaux_wav2med_file2_history_freq_option"`) or a better description (or, ideally, both).